src/scikit_build_core/build/wheel.py

```diff
@@ -214,8 +215,20 @@ def _build_wheel_impl(
 
             process_script_dir(wheel_dirs["scripts"])
 
+        license_files = [
+            Path(x)
+            for y in settings.wheel.license_files
+            for x in glob.glob(y, recursive=True)
+        ]
+        if settings.wheel.license_files and not license_files:
+            logger.warning(
+                "No license files found, set wheel.license-files to [] to suppress this warning"
+            )
+
         with WheelWriter(metadata, Path(wheel_directory), tags.as_tags_set()) as wheel:
             wheel.build(wheel_dirs)
+            for license_file in license_files:
+                wheel.write(str(license_file), license_file.name)
```

Comment:
For reference, `hatchling` writes these in the [metadata folder](https://github.com/pypa/hatch/blob/4886f856eceef04d8f8af05201d157b4c92a0671/backend/src/hatchling/builders/wheel.py#L63) (`f'{project_id}.dist-info'` (PEP427?)).
I.e.: [`f'{project_id}.dist-info/license/{license_file}'`](https://github.com/pypa/hatch/blob/4886f856eceef04d8f8af05201d157b4c92a0671/backend/src/hatchling/builders/wheel.py#L563-L567)

Reply:
Ahh, I thought something was weird when I was setting up the tests, but forgot to look into it. I forgot to prefix this with distinfo, I think.

Interesting, setuptools puts them in `*.dist-info` directly.